### ISSUE TITLE
Idea: add a "beta" app to the play store

### ISSUE THREAD

**Issue by iphydf** [evilcorpltd/aTox#1246]
Upload beta versions there more frequently with less stability/performance guarantee.

**Comment by robinlinden** [comment]
Yeah, we can add a beta track to the GPlay release and allow people to opt into it. Right now the only option for semi-technical users to test newer versions of aTox is the "nightlies" that are attached to every CI run, but they're debug builds, require a GitHub account, and aren't super easy to install, even if you can run them in parallel with a "normal" released aTox.
